File: check_db.js
require('dotenv').config({ path: '.env.local' });
const { createClient } = require('@supabase/supabase-js');

const supabase = createClient(
  process.env.NEXT_PUBLIC_SUPABASE_URL,
  process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY
);

async function checkDb() {
  const { data: users, error: err1 } = await supabase.from('users').select('*');
  console.log('--- USERS ---');
  console.log(users || err1);

  const { data: tickets, error: err2 } = await supabase.from('tickets').select('*');
  console.log('--- TICKETS ---');
  console.log(tickets || err2);

  const { data: kiosk, error: err3 } = await supabase.from('kiosk_settings').select('*');
  console.log('--- KIOSK SETTINGS ---');
  console.log(kiosk || err3);
}

checkDb();

File: check_db.py
import os
import requests

SUPABASE_URL = os.environ.get("NEXT_PUBLIC_SUPABASE_URL")
SUPABASE_KEY = os.environ.get("NEXT_PUBLIC_SUPABASE_ANON_KEY")

headers = {
    "apikey": SUPABASE_KEY,
    "Authorization": f"Bearer {SUPABASE_KEY}",
    "Content-Type": "application/json"
}

def get_table(table):
    res = requests.get(f"{SUPABASE_URL}/rest/v1/{table}?select=*", headers=headers)
    print(f"\n--- Table: {table} ---")
    if res.status_code == 200:
        print(res.json())
    else:
        print(f"Error: {res.status_code} - {res.text}")

get_table("users")
get_table("tickets")
get_table("kiosk_settings")
get_table("transactions")
